app/main.py

```diff
@@ -1 +1,30 @@
-# write your code here
+class Animal:
+    alive = []
+
+    def __init__(self,
```

Comment:
write the first argument after the opening bracket on a new line, and the closing bracket also on a new line